biblio.bib

```diff
@@ -4425,6 +4425,21 @@ @InCollection{FisDhaJou2015lion
   keywords =     {irace},
 }
 
+@InProceedings{FitMalOSuTie2014,
+  author       = "Tadhg Fitzgerald" #and# Malitsky_Y #and# OSullivan_B #and# Tierney,
+  title        = {ReACT: Real-Time Algorithm Configuration through Tournaments},
```

Comment:
You have to protect ReACT from being changed by bibtex styles that change the case of titles. Use {{ReACT}: Real-Time....}